src/Foundation/Commerce/code/Mappers/Profiles/AddressProfile.cs

```diff
@@ -34,6 +34,15 @@ public AddressProfile()
                 .ForMember(dest => dest.Company, opt => opt.Ignore())
                 .ForMember(dest => dest.PhoneNumber, opt => opt.Ignore())
                 .ForMember(dest => dest.Facet, opt => opt.Ignore());
+
+            this.CreateMap<Address, Sitecore.Commerce.Entities.Party>()
+                .ForSourceMember(address => address.Name, opt => opt.Ignore())
+                .ForSourceMember(address => address.CountryCode, opt => opt.Ignore());
+
+            this.CreateMap<Sitecore.Commerce.Entities.Party, Address>()
+                .ForSourceMember(dest => dest.Company, opt => opt.Ignore())
+                .ForSourceMember(dest => dest.PhoneNumber, opt => opt.Ignore())
+                .ForSourceMember(dest => dest.Facet, opt => opt.Ignore());
```

Comment:
How do you map `RegionName` in `CommerceParty` or this is not necessary? (see `UpdateCommerceParty` method in `AccountRepository`)

Reply:
Yea, I missed that, thanks